io.spectra.sudoku_solver.json

```diff
@@ -0,0 +1,51 @@
+{
+    "app-id" : "io.spectra.sudoku_solver",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "43",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command" : "sudoku-solver",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland"
+    ],
+    "build-options" : {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin",
+        "build-args" : [
+            "--share=network"
+        ],
```

Comment:
Flathub builds are done offline. You can use cargo generator to generate part of your manifest: https://github.com/flatpak/flatpak-builder-tools/tree/master/cargo

Reply:
Changed in [ec879b534fada8121c8fec6b2](https://github.com/cyberphantom52/flathub/commit/ec879b534fada8121c8fec6b2fd96280586bc616)